contactsummary/scss/contactsummary.scss

```diff
@@ -2,6 +2,7 @@
 @import "SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/variables";
 
 #bootstrap-theme {
+  @import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/leave-widget";
```

Comment:
just because of this i would be in favour for contactsummary to be the owner of the widget. I mean, havng also an hard coupling between css is yet another bad thing

Reply:
The hard coupling would have to come from somewhere with that approach as well. If the owner is the Contact Summary then we'd need to import a lot of services from L&A (AbsencePeriod, AbsenceType, Entitlement, LeaveRequest, and WorkPattern).